jobclient/java/src/test/java/com/twosigma/cook/jobclient/JobTest.java

```diff
@@ -47,6 +47,7 @@
     // Constraints which could be used for any test.
     private Constraint _constraint1;
     private Constraint _constraint2;
+    private static final double DELTA = 1e-15;
```

Comment:
usually called EPSILON (don't have to change though)

Reply:
ah yeah, changed